cypress/integration/trends_elements.js

```diff
@@ -100,7 +100,7 @@ describe('Trends actions & events', () => {
     it('Save to dashboard', () => {
         cy.get('[data-attr=save-to-dashboard-button]').click()
         cy.contains('Add panel to dashboard').click()
-        cy.wait(500) // not ideal but toast has a delay render
+        cy.wait(700) // not ideal but toast has a delay render
         cy.get('[data-attr=success-toast]').should('exist')
```

Comment:
Should this not wait already? Is the wait above to avoid timing out?

Reply:
cypress should wait when i originally wrote these tests it wasn't catching the toast without the wait